test/integration/main_test.rb

```diff
@@ -19,7 +25,7 @@ class MainTest < IntegrationTest
 
     kamal :redeploy
     assert_app_is_up version: second_version
-    assert_hooks_ran "pre-connect", "pre-build", "pre-deploy", "post-deploy"
+    assert_hooks_ran "pre-connect", "pre-deploy", "post-deploy"
```

Comment:
Does this not call the `pre-build` hook? I would expect it to because we are deploying a new version here.